packages/post-purchase-ui-extensions/src/extension-points/api/post-purchase/post-purchase.ts

```diff
@@ -240,22 +270,34 @@ interface CalculatedPurchase {
   totalOutstandingSet: MoneyBag;
 }
 
-/** Represents an error occurred while calculating or applying a changeset.
+/** Error code while calculating or applying a changeset.
  *
- * **Possible values for `code`:**
+ * **Possible values :**
  *
  * - `payment_required` - The original payment method couldn't be charged.
  * - `insufficient_inventory` - An item requested to be added is out of stock.
  * - `changeset_already_applied` - Indicates if the changeset has been applied. Changesets can only be applied once.
  * - `unsupported_payment_method` - Indicates that the payment method was unsupported. The payment method used to pay for the initial purchase can't be used to pay for changesets.
  * - `invalid_request` - The request is invalid. It shouldn't be re-sent without modifications.
  * - `server_error` - An unexpected error has happened.
+ * - `buyer_consent_required` - Indicates that the buyer's consent was required in order to apply the changeset, but none was provided.
+ */
+type ChangesetErrorCode =
+  | 'payment_required'
+  | 'insufficient_inventory'
+  | 'changeset_already_applied'
+  | 'unsupported_payment_method'
+  | 'invalid_request'
+  | 'server_error'
+  | 'buyer_consent_required';
```

Comment:
👍  I'll write a reminder to myself to come back to change this to add the new error added by the contract creation PR. Perhaps the subs vaulting PR is also adding some.

Reply:
Feel free to add it to this PR, lets merge this until we have all of the new types reflecting the new APIs.